docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -150,21 +157,26 @@ There is also a method to export the whole project into specific export formats.
 
 ### The `.export_labels` method
 
-The `.export_labels` method enables the export of a full project. It does the following preprocessing:
+The [`.export_labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.export_labels) method enables the export of a full project. It does the following preprocessing:
 
 * Only fetches the labels of types `"DEFAULT"` and `"REVIEW"` (see the [label types explanations](https://docs.kili-technology.com/docs/asset-lifecycle#label-types-and-definitions-throughout-an-asset-lifecycle)).
 * If specified, selects a subset of asset ids.
 * Exports labels to one of the standard formats (only available for a restricted set of ML tasks).
-* The `with_assets` argument lets you decide if you want to include the assets in the export.
-* The `export_type` argument tells if the latest label or all the labels are exported.
-* The `split_option` argument tells if the export contains one folder for all the jobs, or one folder per job.
-* The `single_file` argument tells if the labels data should be exported into one single file. Note that some formats are single file or multiple files only:
-     * Kili: single file or multiple files.
-     * YOLO: multiple files only.
-     * Pascal VOC: multiple files only.
-     * COCO: single-file only.
-
-For all the formats, in the output archive, a README.kili.txt file is also created. Here is an example of its contents:
+* Using various method arguments, you can decide:
+    * Whether or not to include the assets in the export
+    * Whether to export just the latest label or all the labels
+    * Whether to create one folder for all the jobs or one folder per job
+    * Whether or not to export the label-related data into one single file
```

Comment:
we should keep the argument names here I think

Reply:
But the link to the method is already here: 

>The [`.export_labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.export_labels) method enables the export of a full project.